### DISCUSSION TITLE
SadTalker doesn't show up as a new tab

### DISCUSSION THREAD

**Discussion by githubguy2024** [OpenTalker/SadTalker#858] [General]
![sadtalker launching error](https://github.com/OpenTalker/SadTalker/assets/163620851/89f95e6e-0e4c-4b05-9fed-0f60f31dbd3c)

I installed Sadtalker with several methods and everything went correctly, but it doesn't show up as a new tab. on cmd, after double tab on webui.bat, before launching the extension the error on the screenshot appears.
BTW, I tried installing Python 3.10.6 and now 3.10.9.
somebody help please.

**Comment by terrysoo** [comment]
The script "app_sadtalker.py" is compatible with gradio verison 3.50.0. You might have installed a higher version (like 4.27.0). If yes, run "pip install gradio==3.50.0", and try again. Good luck!
